documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -741,7 +763,104 @@ The topic is named according to the pattern `<heartbeat.topics.prefix>.<server.n
 
 |`sanitize.field.names`
```

Comment:
Should this configuration option simply not exist in the downstream documentation if Avro isn't supported?

Reply:
I would leave it in, as eventually usage with the Apicurio registry will be supported.